test/ueberauth_oidc/strategy/oidc_test.exs

```diff
@@ -319,6 +319,27 @@ defmodule Ueberauth.Strategy.OIDCTest do
              } = OIDC.info(conn)
     end
 
+    test "Get info from the conn and ignore invalid fields" do
+      info =
+        OIDC.info(%Plug.Conn{
+          private: %{
+            ueberauth_oidc_user_info: %{
+              "name" => "name",
+              "foo" => "bar"
+            }
+          }
+        })
+
+      refute Map.has_key?(info, :foo)
```

Comment:
Not really needed since Info doesn't define this field, but just for extra explicitness :)